Makefile

```diff
@@ -93,7 +93,7 @@ tar: doc
 
 doc/%.1 doc/%.7: doc/%.txt version.mk
 	$(call colorecho,DOC,$@)
-	$(VERBOSE) $(A2X) -f manpage -a "herbstluftwmversion=herbstluftwm $(VERSION)" -a "date=`date +%Y-%m-%d`" $<
+	$(VERBOSE) $(A2X) -f manpage -a "herbstluftwmversion=herbstluftwm $(VERSION)" -a "date=`date -u -r NEWS %Y-%m-%d`" $<
```

Comment:
When using a file's modification date, I think one should use `$<` because the modification date of `NEWS` is independent from the man page modification date.

Reply:
I'm wrong for released versions. I think it is fine as it is!